<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Eliot</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../eliot.qrc" >
    <normaloff>:/images/eliot.xpm</normaloff>:/images/eliot.xpm</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout_2" >
    <item>
     <widget class="QSplitter" name="splitterHoriz" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitterVert" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QGroupBox" name="groupBoxTest" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>_("Board")</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBoxHistory" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>_("History")</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBoxPlayers" >
       <property name="title" >
        <string>_("Players")</string>
       </property>
       <layout class="QHBoxLayout" />
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>splitterHoriz</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="contextMenuPolicy" >
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_WindowsBag" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>_("Bag")</string>
   </property>
   <property name="statusTip" >
    <string>_("Show/hide the remaining tiles in the bag")</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_WindowsBoard" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>_("External board")</string>
   </property>
   <property name="statusTip" >
    <string>_("Show/hide the external board")</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_WindowsHistory" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>_("History")</string>
   </property>
   <property name="statusTip" >
    <string>_("Show/hide the game history")</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_WindowsDicTools" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>_("Dictionary tools")</string>
   </property>
   <property name="statusTip" >
    <string>_("Show/hide the dictionary tools")</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../eliot.qrc" />
 </resources>
 <connections/>
</ui>
